Packs/Illumio/Integrations/IllumioCore/IllumioCore_test.py

```diff
@@ -1,17 +1,17 @@
 """Test file for Illumio Integration."""
 
 import io
-import json
+
+import illumio.pce
 import pytest
-from illumio import PolicyComputeEngine, IllumioException
-from CommonServerPython import *  # noqa
-from IllumioCore import InvalidMultiSelectException, InvalidSingleSelectException, InvalidPortException, \
-    VALID_POLICY_DECISIONS, VALID_PROTOCOLS
 
+from CommonServerPython import *  # noqa
+from IllumioCore import *
+from test_data.test_data_responses import *
```

Comment:
Ok, again, * imports should be avoided where possible. And if they must be used, you should always define `__all__` so that you aren't also importing the module's dependencies.

Similarly, don't use `import illumio.pce` as above. You're just setting yourself up for name conflicts.

Reply:
Okay, will import only necessary things from `test_data`.